index.bs

```diff
@@ -6539,6 +6539,44 @@ A <dfn id="dfn-frozen-array-type" export>frozen array type</dfn> is a parameteri
 type whose values are references to objects that hold a fixed length array
 of unmodifiable values.  The values in the array are of type |T|.
 
+Frozen array types must only be used as the type of [=regular attributes=] or [=static attributes=]
+defined on an [=interface=].
+
+<div class="example" id="example-frozen-array">
+    The following [=IDL fragment=] defines an [=interface=] with two frozen array attributes, one
+    [=read only=] and one not.
+
+    <pre highlight="webidl">
+        [Exposed=Window]
+        interface PersonalPreferences {
+            readonly attribute FrozenArray&lt;DOMString> favoriteColors;
+            attribute FrozenArray&lt;DOMString> favoriteFoods;
+
+            undefined randomizeFavoriteColors();
+        };
+    </pre>
+
+    The behavior of these attributes could be defined like so:
+
+    <blockquote>
+        Each <code>PersonalPreferences</code> has associated <b>favorite colors</b>, a {{FrozenArray}}&lt;{{DOMString}}>, initially equal to the result of [=creating a frozen array=] from « "<code>purple</code>", "<code>aquamarine</code>" ».
+
+        Each <code>PersonalPreferences</code> has an associated <b>favorite foods</b>, a {{FrozenArray}}&lt;{{DOMString}}>, initially equal to the result of [=creating a frozen array=] from the empty list.
+
+        The <code>favoriteColors</code> [=getter steps=] are to return [=this=]'s favorite colors.
+
+        The <code>favoriteFoods</code> [=getter steps=] are to return [=this=]'s favorite foods.
+
+        The <code>favoriteFoods</code> [=setter steps=] are to set [=this=]'s favorite foods to [=the given value=].
```

Comment:
This just works? So if it's not frozen it ends up throwing or some such?

Reply:
The conversion steps for JS values to Web IDL `FrozenArray<T>` types will take any iterable, iterate over it, grab the values into an Infra list, then create a JS array from them, then freeze it, then package it up into a `FrozenArray<T>` pointer. So yeah, it just works.